com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/IBannerEventListener.kt

```diff
@@ -4,8 +4,8 @@ package com.chartboost.heliumsdk.unity
  * Interface for sending real-time Banner events
  */
 interface IBannerEventListener {
-    fun DidLoadBanner(placementName: String, errorCode: Int, errorDescription: String)
-    fun DidClickBanner(placementName: String, errorCode: Int, errorDescription: String)
-    fun DidRecordImpression(placementName: String, errorCode: Int, errorDescription: String)
-    fun DidWinBidBanner(placementName: String, auctionId: String, partnerId: String, price: Double)
+    fun DidLoadBanner(placementName: String, loadId: String, error: String)
```

Comment:
my recommendation is to combine this and put the did win information in the did load callback